.github/workflows/build.yml

```diff
@@ -24,11 +24,27 @@ jobs:
     - name: Npm Install
       run: npm install
 
-    - name: Lint
-      run: npm run lint
-
     - name: Build
-      run: npm run build
+      run: npm run build -- -- -- --skip-lint
 
     - name: Test
-      run: npm run test
+      run: npm run test -- -- -- --skip-lint
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+
+    - name: Npm Install Lerna
+      run: npm install -g lerna
```

Comment:
Why is this required? Npm install should install lerna locally and script that use it should be able to reference it.

Reply:
I dunno, I copied this from what was already there. I assume this logic holds for both lint and build? I'll try removing it and seeing what happens but probably not today. Thanks for the quick review :)

Reply:
I took this step out of both jobs and everything still worked correctly, thanks!